src/elexmodel/handlers/data/Estimandizer.py

```diff
@@ -0,0 +1,111 @@
+class Estimandizer:
+    """
+    Estimandizer. Generate estimands explicitly.
+    """
+
+    def __init__(self, data_handler, office, estimands={}):
+        self.data_handler = data_handler
+        self.office = office
+        self.estimands = estimands
+        self.transformations = []
+        self.transformation_map = {
+            "party_vote_share": [self.calculate_party_vote_share],
+            "candidate": [self.candidate],
+        }
+
+    def pre_check_estimands(self):
+        """
+        Ensure estimand isn't one of the pre-specified values that are already included
+        """
+        standard = ["dem_votes", "gop_votes", "total_votes"]
+        if not self.check_input_columns(standard):
+            self.create_estimand(None, self.standard)
+
+    def check_input_columns(self, columns):
+        """
+        Check that input columns contain all neccessary values for a calculation
+        """
+        missing_columns = []
+        if self.office == "G":
+            missing_columns = [col for col in columns if col not in self.data_handler.data.columns]
+        elif self.office == "P":
+            missing_columns = [
+                col for col in columns if col not in self.data_handler.data[self.data_handler.election_id]
```

Comment:
why is there a difference in governor and presidential elections?

Reply:
Same as above, I meant to look at election_type since they have different input structures (general vs primary)- fixed.